Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -33,6 +33,9 @@
 import demistomock as demisto
 import warnings
 
+MAX_MSG_LENGTH = 60000
+MAX_CHARS_PREFIX_SUFFIX = 10000  # for shortening a long message
```

Comment:
Since there's 7,300 lines of code between this and the only place where it's used, let's define it only there (in line 7303)